indico/MaKaC/PDFinterface/base.py

```diff
@@ -708,8 +709,21 @@ def __init__(self):
         latex_mdx = mdx_latex.LaTeXExtension()
         latex_mdx.extendMarkdown(md, markdown.__dict__)
 
+        # Escape LaTeX before markdown
+        def _convert_markdown(text):
+            math_segments = []
+            placeholder = u"\uE000"
+
+            def _math_replace(m):
+                math_segments.append(m.group(0))
+                return placeholder
+
+            text = re.sub(r'\$[^\$]+\$|\$\$(^\$)\$\$', _math_replace, text)
```

Comment:
Looks like the `LaTeXExtension.extendMarkdown` method is not escaping latex. However I totally missed the `latex_escape` function. I will see if using it fixes the problem. Thanks for the observation.